packages/lib/modules/tokens/TokenIconStack.tsx

```diff
@@ -40,7 +40,7 @@ export function TokenIconStack({
             borderColor="background.base"
             borderRadius="100%"
             height={`${size + 4}px`}
-            key={tokenAddress}
+            key={tokenAddress + i}
```

Comment:
assuming you added index because `tokenAddress` could be duped in some situation but im having trouble imagining scenario that would be true. 

could you share example?

Reply:
It was showing errors on console. The problem can be either due to showing the underlying token on screen and having a couple of tokens with the same one or maybe just the token address being undefined for a small period of time before stabilizing.